ocamlformat-rpc.opam

```diff
@@ -10,24 +10,9 @@ bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
 depends: [
   "dune" {>= "2.8"}
   "ocaml" {>= "4.08" & < "4.15"}
-  "ocamlformat-rpc-lib"
   "alcotest" {with-test}
-  "base" {>= "v0.12.0" & < "v0.15"}
-  "cmdliner" {>= "1.1.0"}
-  "dune-build-info"
-  "either"
-  "fix"
-  "fpath"
-  "menhir" {>= "20201216"}
-  "menhirLib" {>= "20201216"}
-  "menhirSdk" {>= "20201216"}
-  "ocaml-version" {>= "3.3.0"}
-  "ocp-indent"
-  "odoc-parser" {>= "1.0.0"}
-  "re" {>= "1.7.2"}
-  "stdio" {< "v0.15"}
-  "uuseg" {>= "10.0.0"}
-  "uutf" {>= "1.0.1"}
+  "ocamlformat" {= version}
```

Comment:
Why should `ocamlformat-rpc` have the same version as `ocamlformat`? This is making some dependencies problem... (see https://github.com/ocaml/ocaml-lsp/pull/599#issuecomment-1083375199)

Reply:
`ocamlformat-rpc` was the rpc server, it is specific to the version of ocamlformat. Since recently, the RPC server is part of the `ocamlformat` package, no more confusion.
I think you meant to use `ocamlformat-rpc-lib`, which is the library for defining a client and doesn't depends on ocamlformat.

Reply:
Okay. I did meant `ocamlformat-rpc`, as this breaks the `ocaml-lsp` CI (`ocaml-lsp` CI needs `ocamlformat-rpc.0.21.0` (for testing) and `ocamlformat.0.19.0` (for checking formatting). But I guess that's logic to bound those version, I'm sure it can be worked out in ocaml-lsp CI :)